.vscode/tasks.json

```
@@ -2,34 +2,34 @@
   "version": "2.0.0",
   "tasks": [
     {
-      "label": "Build database",
+      "label": "Run Evidence",
       "type": "shell",
-      "command": "uv run pipelines/run.py run build_database",
+      "command": "if command -v npm > /dev/null 2>&1; then cd analytics/evidence && npm install && npm run sources && npm run dev -- --host 0.0.0.0; else echo 'NPM not installed'; fi",
+      "windows": {
+        "command": "cd analytics\\evidence; npm install; npm run sources; npm run dev -- --host 0.0.0.0"
+      },
       "group": "none",
       "icon": {
-        "id": "database"
+        "id": "run"
       },
+      "problemMatcher": null,
       "presentation": {
         "reveal": "always",
         "panel": "new"
       }
     },
     {
-      "label": "Run Evidence",
+      "label": "Download database",
       "type": "shell",
-      "command": "if command -v npm > /dev/null 2>&1; then cd analytics/evidence && npm install && npm run sources && npm run dev -- --host 0.0.0.0; else echo 'NPM not installed'; fi",
-      "windows": {
-        "command": "cd analytics\\evidence; npm install; npm run sources; npm run dev -- --host 0.0.0.0"
-      },
+      "command": "uv run pipelines/run.py run download_database",
```

```suggestion
      "command": "uv run pipelines/run.py run download_database_https",
```
On veut utiliser le download via HTTPS :) 
On peut aussi mettre explicitement l'option env pour que ce soit plus clair. Ou alors il y a peut être une façon de le demander via l'UI de vscode.

---

J'avais pas vu qu'il y avait une fonction _https ! Pour l'environnement, du coup tu veux prod ou dev ?

---

> il y a peut être une façon de le demander via l'UI de vscode.

J'ai regardé dans la [documentation](https://marketplace.visualstudio.com/items?itemName=actboy168.tasks), et j'ai pas trouvé. Donc je vais faire prod comme a suggéré @jereze, à savoir l'option `prod`.